### ISSUE TITLE
Make Mensa converge in multi-event scenarios

### ISSUE THREAD

**Issue by thecml** [thecml/mensa#25]
The model does not converge for multi-event scenarios. Probably has to do with the loss function or the copula.

Training script: https://github.com/thecml/mensa/blob/main/src/experiments/run_synthetic_multi_event.py
